### ISSUE TITLE
Wrong Charset Broke Typesense .NET Client

### ISSUE THREAD

**Issue by HazyFish** [typesense/typesense#1166]
## Description

Charset should be "utf-8" not "utf8".

https://github.com/typesense/typesense/blob/5f563184c1be8df7f66a88982c39a8e1b5541774/src/core_api.cpp#L732

@kishorenc 

## Expected Behavior

Typesense v0.25.0 should work with .NET Typesense Client (https://github.com/DAXGRID/typesense-dotnet)

## Actual Bahavior

Integration Test on Export failed when using Typesense v0.25.0.

```
System.InvalidOperationException
  HResult=0x80131509
  Message=The character set provided in ContentType is invalid. Cannot read content as string using an invalid character set.
  Source=System.Net.Http
  StackTrace:
   at System.Net.Http.HttpContent.ReadBufferAsString(ArraySegment`1 buffer, HttpContentHeaders headers)

  This exception was originally thrown at this call stack:
    System.Text.EncodingTable.InternalGetCodePageFromName(string)
    System.Text.EncodingTable.GetCodePageFromName(string)
    System.Text.Encoding.GetEncoding(string)
    System.Net.Http.HttpContent.ReadBufferAsString(System.ArraySegment<byte>, System.Net.Http.Headers.HttpContentHeaders)

Inner Exception 1:
ArgumentException: 'utf8' is not a supported encoding name. For information on defining a custom encoding, see the documentation for the Encoding.RegisterProvider method. (Parameter 'name')
```

## Steps to Reproduce

Run integration tests for .NET Typesense Client (https://github.com/DAXGRID/typesense-dotnet) against Typesense v0.25.0.

**Comment by kishorenc** [comment]
My bad, thanks for reporting this. I've already fixed it, and will be rolling out some of these smaller bug fixes as a 0.25.1 release next week.

**Comment by HazyFish** [comment]
I can confirm it is fixed! Thank you!!!
